tutorials/timefreq/plot_MorletWavelet.py

```diff
@@ -0,0 +1,127 @@
+"""
+Morlet Wavelet Analysis
+===============
+
+Perform time-frequency decomposition using wavelets.
+
+This tutorial will explore the ``neurodsp.timefrequency.wavelets`` module.
+It uses Morlet wavelets to transform time series data to a time-frequency
+representation of the data.
+"""
+
+###################################################################################################
+
+# Import neccessary functions and packages
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.signal import morlet
+
+from neurodsp.utils.data import create_freqs
+from neurodsp.utils.checks import check_n_cycles
+from neurodsp.utils.decorators import multidim
+from neurodsp.filt.filter import filter_signal
+
+# Import simulation code to create test Data
+from neurodsp.sim import sim_combined, sim_bursty_oscillation
+from neurodsp.sim.cycles import sim_cycle
+from neurodsp.utils import set_random_seed, create_times
+
+# Import utilities for loading and plotting data
+from neurodsp.utils.download import load_ndsp_data
+from neurodsp.plts.time_series import plot_time_series
+
+# Import functions for morlet Wavelets
+from neurodsp.timefrequency.wavelets import compute_wavelet_transform, convolve_wavelet
+
+###################################################################################################
+
+# Set the random seed, for consistency simulating data
+set_random_seed(0)
+
+###################################################################################################
+# Simulate time-frequency series data
+# -----------------------------
+#
+# First, we'll simulate the time frequency series using the function 'sim_bursty_oscillation'. This will simulate time-varying oscillations.
```

Comment:
Are these too long? It looks weird with the `#`s being much shorter than the text, usually the text stays within them but I believe the lines just have to be under 100 characters

Reply:
quick note since I noticed this comment: 
In terms of rendering the website, it doesn't really matter how long the text lines are. By convention, and for readability of the source files, we do tend to keep text lines under 100 characters. Notably, as long as you continue on the next line, a line break in the file doesn't enforce a line break on the render, so adding line breaks doesn't make the sphinx page go weird (you can compare the layout of existing tutorial files, and what they look like on the website). 
tl-dr: best practice is to keep text lines < 100 chars.

Also, in line 42/43 (and any similar cases): lengths of the title underhang (---) should match length of the title

Reply:
Thanks!